eZ/Publish/Core/Search/Legacy/Tests/Content/Location/HandlerSortTest.php

```diff
@@ -124,6 +129,34 @@ protected function getLocationSearchHandler()
         );
     }
 
+    protected $contentTypeGateway;
```

Comment:
Not really a point on this PR itself, but this introduces more duplicated code, there should be some inheritance reuse from equal content to location based search tests and implementations (see also Sensio Insights reports, a lot of duplications comes from Content/Locations search split)